packages/discord.js/src/structures/Message.js

```diff
@@ -30,127 +30,401 @@ const Util = require('../util/Util');
  * @extends {Base}
  */
 class Message extends Base {
-  constructor(client, data) {
+  constructor(client, data = {}) {
     super(client);
+    this.data.channelId = data.channel_id;
+    this.data.guildId = data.guild_id ?? this.channel?.guild?.id ?? null;
```

Comment:
data should stay snake_case probably, also you're not getting the camelCase versions later

Reply:
Agreed, as message has guildId & channelId getters, there's no real purpose of storing camelCase properties.

Reply:
Sorry, I thought I fixed this don't know why I marked this as resolved.